les/odr_test.go

```diff
@@ -135,9 +135,8 @@ func odrContractCall(ctx context.Context, db ethdb.Database, config *params.Chai
 				context := core.NewEVMContext(msg, header, bc, nil)
 				vmenv := vm.NewEVM(context, statedb, config, vm.Config{})
 
-				//vmenv := core.NewEnv(statedb, config, bc, msg, header, vm.Config{})
 				gp := new(core.GasPool).AddGas(math.MaxUint64)
-				result, _ := core.ApplyMessage(vmenv, msg, gp)
+				result, _ := core.ApplyMessage(vmenv, msg, gp, nil)
```

Comment:
More places where we use nil.  Again, I would prefer using a fake evm runner if it allows us to remove the handling for nil in GetGasPriceMinimum.  Same with the one 11 lines below this one.

Reply:
using celomock runner now